docs/src/man/guide.md

```diff
@@ -2,134 +2,123 @@
 
 Documenter is designed to do one thing -- combine markdown files and inline docstrings from
 Julia's docsystem into a single inter-linked document. What follows is a step-by-step guide
-to creating a simple document.
+intended for users who iteratively develop a package and write documentation for it.
 
+## Preliminaries
 
-## Installation
-
-Documenter can be installed using the Julia package manager.
-From the Julia REPL, type `]` to enter the Pkg REPL mode and run
+As a package developer you want to keep a single Julia REPL session opened from which you
+generate your docs. To make Julia aware of changes to your code and docstrings, you need to
+install [Revise.jl](https://github.com/timholy/Revise.jl) first. Enter the Pkg REPL by
+pressing `]` and run:
 
 ```
-pkg> add Documenter
+(@v1.5) pkg> add Revise
+
+julia> using Revise
```

Comment:
Might be good to warn here to make sure Revise gets added to the _global_ environment (even though it's implied by `(@1.5)>`), and not to either as a package dependency or in `docs/Project.toml`.